packages/react/index.js

```diff
@@ -1,6 +1,36 @@
 'use strict';
 
+function testMinification() {
+  if (testMinification.name === 'testMinification') {
+    // We are not minified.
+    // This might be a Node environment where DCE is not expected anyway.
+    return;
+  }
+  if (process.env.NODE_ENV === 'production') {
```

Comment:
This outer wrapper is probably unnecessary since we are already calling it from the same condition.